cmd/ipsw/cmd/download/download_ipsw.go

```diff
@@ -312,8 +312,8 @@ var ipswCmd = &cobra.Command{
 					if err != nil {
 						log.Errorf("failed to get folder from remote ipsw metadata: %v", err)
 					}
-					destPath = filepath.Join(destPath, folder)
-					if err := utils.RemoteUnzip(zr.File, dlRE, destPath, flat); err != nil {
+					var folderDestPath = filepath.Join(destPath, folder)
```

Comment:
it looks like I don't use the var anywhere else before the prog returns so we don't need a variable and can just use it as an arg to RemoteUnzip

Reply:
Applied your suggestions :)